packages/tdesign-react/site/plugins/changelog-to-json.js

```diff
@@ -0,0 +1,308 @@
+import { promises, readdirSync, statSync } from 'fs';
+import path from 'path';
+
+const targetRoot = path.join(process.cwd(), '../tdesign-react');
+const outputPath = path.join(targetRoot, '../site/dist/changelog.json');
+const markdownPath = path.join(targetRoot, '../CHANGELOG.md');
+const componentsDir = path.join(targetRoot, '../../components');
+const excludedDir = ['_util', 'common', 'hooks', 'locale', 'style'];
+
+const LOG_TYPES = ['🚀 Features', '🐞 Bug Fixes', '❗ Breaking Changes'];
```

Comment:
```suggestion
const LOG_TYPES = ['❗ Breaking Changes', '🚀 Features', '🐞 Bug Fixes' ];
```
顺序
1 Breaking Changes
2 Features
3 Bug Fixe